common/djangoapps/student/urls.py

```diff
@@ -30,6 +30,15 @@
 
     url(r'^change_setting$', 'change_setting', name='change_setting'),
     url(r'^change_email_settings$', 'change_email_settings', name='change_email_settings'),
+
+    # password reset in student.views (see below for password reset django views)
```

Comment:
Reviewers: Moving this up here has the advantage of keeping urls with `student.views` as default together, but moves this further from the other reset password url(s).